content/working_with/console/widgets/deployments.md

```diff
@@ -8,8 +8,8 @@ Displays the list of the deployments in the current tenant, according to the use
 
 ![blueprint-deployments]( /images/ui/widgets/blueprint-deployments.png )
 
-Each deployment in the list includes details about the attached blueprint, when the deployment was created (if it was updated icon indicating that will be shown near creation date),
-which site it is assigned to and the nodes on which it is deployed.
+Each deployment in the list includes details about the attached blueprint, when the deployment was created (if it was updated icon indicating that will be shown near creation date)
```

Comment:
If I remember correctly Site Name is presented, but only when it is set.

Reply:
Right - reverted.